docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+|Team agility| An issue targeted to reduce friction to releasing new versions of the SDKs.| ![#fbca04](https://via.placeholder.com/15/fbca04/000000?text=+) `#fbca04` | `Type:Team agility` |
+
+### Size
+
+Use these to assign an estimated level of effort to resolve an issue and assist with the estimation process.
+
+Color: All the labels in this category should use ![#91e3ea](https://via.placeholder.com/15/91e3ea/000000?text=+) `#91e3ea`.
+
+|Name| Description  | Example
+|---|---|------|
+|Small| The issue is simple and well understood, it will take a day or less to complete | `Size:S` |
+|Medium| The issue is not very complex and it is well understood, it will take 1 to 3 days to complete | `Size:M` |
+|Large| The issue complex but it is well understood, it will take 4 to 8 days to complete | `Size:L` |
+|Extra Large| The issue very complex or not very well defined, it will take 9 to 14 days to complete. In this case, it is probably better to rethink the issue and break it down in smaller tasks | `Size:XL` |
+
+### Automation
+
+These labels are applied to PRs and used to trigger or disable github workflows.
+
+Color: All the labels in this category should use ![#cccccc](https://via.placeholder.com/15/cccccc/000000?text=+) `#cccccc`.
+
+|Name| Description  | Example
+|---|---|------|
+|no parity| PR does not need to be applied to other languages. **Note:** if you don't apply the `No parity` to a dotnet PR, the automation workflow will generate parity issues in Python, JS and Java.  | `Automation:No parity` |
+|parity with dotnet| The PR needs to be ported to dotnet. | `Automation:Parity with dotnet` |
+|parity with JS| The PR needs to be ported to JS. | `Automation:Parity with JS` |
+|parity with Python| The PR needs to be  ported to Python. | `Automation:Parity with JS` |
+|parity with Java| The PR needs to be ported to Java. | `Automation:Parity with Java` |
+
+### DRI
+
+DRI labels are used for reporting outside the bot framework repositories and their names don't follow the standards described above. They should be applied based on the current DRI guide.
+
+Color: This subcategory uses different colors for each label.
+
+|Name| Description  | Color | Example
+|---|---|---|------|
+|Bot Services| Required for internal Azure reporting. Do not delete. Do not change color.| ![#e99695](https://via.placeholder.com/15/e99695/000000?text=+) `#e99695` | `Bot Services` |
+|customer-reported| Customer reported issues.| ![#c2e0c6](https://via.placeholder.com/15/c2e0c6/000000?text=+) `#c2e0c6` | `customer-reported` |
+|customer-replied-to| Required for internal Azure reporting. Do not delete.| ![#2683a5](https://via.placeholder.com/15/2683a5/000000?text=+) `#2683a5` | `customer-replied-to` |
+|ExemptFromDailyDRIReport| Use this label to exclude the issue from the DRI report.| ![#bde567](https://via.placeholder.com/15/bde567/000000?text=+) `#bde567` | `ExemptFromDailyDRIReport` |
```

Comment:
If we are controlling this label, can we use the same naming mechanism as above?

Reply:
These labels are imposed by the DRI process, I don't think we can change them but I will ask.